### DISCUSSION TITLE
Adding alias

### DISCUSSION THREAD

**Discussion by ajitbohra** [tailwindlabs/tailwindcss#1796] [Help]
What's the best way to add the alias for generated classes.

e.g. `has-background-white` for `bg-white` 

helpful when classes are dynamically generated and cannot be controlled.

one way would be 

`.has-background-white {
	@apply: bg-white
}
`

but this makes things bit redundant

**Comment by wongjn** [comment]
You could write a custom plugin that emulates the `bg-` classes. However, the classes that are "dynamically generated and cannot be controlled" would need to be present somewhere to be scanned by Tailwind at build time for these classes to be generated.
